File: R/app/src/main/java/com/example/voiceassistent/numberapi/Number.java
package com.example.voiceassistent.numberapi;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Number implements Serializable {
    @SerializedName("str")
    @Expose
    public String current;
}

File: Rkt/app/src/main/java/com/example/voiceassistent/numberapi/Number.kt
package com.example.voiceassistent.numberapi

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Number : Serializable {
    @JvmField
    @SerializedName("str")
    @Expose
    var current: String? = null
}